README.md

```diff
@@ -276,11 +285,14 @@ after answering. See the example below.
 
 ### 2. Feedback questionnaire
 
-A feedback questionnaire is basically just like a graded questionnaire, but with
-the `feedback` option it uses the feedback category and CSS class by default.
+A feedback questionnaire is almost like a graded questionnaire. With
```

Comment:
With the `feedback` option questionnaire uses the feedback category and CSS class by default.
change that to
When the `feedback` option is set, the questionnaire uses the feedback category and CSS class by default.